pyanaconda/ui/tui/simpleline/base.py

```diff
@@ -157,7 +157,7 @@ def _thread_input(self, queue_instance, prompt, hidden):
         """
 
         if hidden:
-            data = self.simpleline_getpass(prompt)
+            data = self.simpleline_getpass = prompt
```

Comment:
This will not work, because `self.simpleline_getpass` is a function. Why is it also a property, I have no idea.